### DISCUSSION TITLE
Options Picker Not Working

### DISCUSSION THREAD

**Discussion by tasneemimamkhan** [Budibase/budibase#1409] [Help] [answered]
Hi! 

Thank you for this phenomenal platform! I am having issues with the "Options Picker" data type under "Forms". Specifically, I'm having trouble creating the dropdown. In the "placeholder" field, what do I enter to create a dropdown? For instance, if I want a dropdown list to look like the following,

A
B
C

what should the delimiter be? I've tried using commas, new lines, handlebar programming, etc, to no success. Any assistance would be greatly appreciated. 

Thank you so much! This is such a fantastic platform, I can't say it enough. 

Warmly,
N

**Comment by mjashanks** [comment, marked as answer]
Thanks!

You must bind your options picker to an "Options" column type. So:

Create a new column, of type options. In here you can choose the available options:
![Screenshot_20210426_113427](https://user-images.githubusercontent.com/3524181/116069882-d20e2100-a683-11eb-8932-4aa88bc303f4.png)

Then, in your options picker, you must set the "Field" property to your new column:
![Screenshot_20210426_113539](https://user-images.githubusercontent.com/3524181/116069973-ece09580-a683-11eb-9b9c-f05ae59008a7.png)

Hope this helps!

Mike

**Reply by tasneemimamkhan** [reply]
Thank you so much for the speedy response! 

For some reason, I don't see that new column in the "Fields" dropdown for the "Option Picker"? I will show you a screenshot of what I mean:

<img width="1097" alt="Screen Shot 2021-04-26 at 3 22 01 PM" src="https://user-images.githubusercontent.com/82778447/116139346-ec57f700-a6a3-11eb-9076-ce73731747ac.png">

<img width="259" alt="Screen Shot 2021-04-26 at 3 24 25 PM" src="https://user-images.githubusercontent.com/82778447/116139386-f679f580-a6a3-11eb-8f58-bfadd3ee27d0.png">

Any help would be appreciated! Thanks so much!!

**Reply by mjashanks** [reply]
No problem!

You also must place your `Options Picker` inside a `Form`, and set the Form's schema to the table where you added the column.

You should read more about forms, field groups and fields here: https://docs.budibase.com/design/forms/form-component

**Reply by tasneemimamkhan** [reply]
IT WORKS!!!!!!! Thank you so much!!!

**Reply by mjashanks** [reply]
You're welcome!
